data/tutorials/getting-started/2_00_editor_setup.md

```diff
@@ -148,9 +148,13 @@ Other Merlin commands for Vim are available and you can checkout their usage on
 
 ## Neovim
 
-Neovim comes with an LSP client.
+Neovim comes with an LSP client for basic features, such as code navigation and diagnostics.
+For the advanced Merlin features, you can also have them by adding the plugin `ocaml.nvim`.
 
-One note here is that is that `ocaml-lsp-server` is sensitive to versioning, and often does not play well with the sometimes outdated sources in Mason, a popular package manager for language services. We recommend you install the LSP server directly in the switch, and pointing your Neovim config to use that.
+### Installing and configuring the LSP
+
+> [!NOTE]
+> One note here is that is that `ocaml-lsp-server` is sensitive to versioning, and often does not play well with the sometimes outdated sources in Mason, a popular package manager for language services. We recommend you install the LSP server directly in the switch, and pointing your Neovim config to use that.
```

Comment:
Do Neovim people actually understand what that means ? What is Mason ?

Reply:
This text is not from me, it was already there. I just added the note style to separate into 1.LSP 2.`ocaml.nvim`.